api/src/Entity/Activity.php

```diff
@@ -75,7 +84,7 @@ class Activity extends BaseEntity implements BelongsToCampInterface {
     )]
     #[ApiProperty(
         writableLink: true,
-        example: '[{ "period": "/periods/1a2b3c4a", "endOffset": 1100, "startOffset": 1000 }]',
+        example: '[{ "period": "/periods/1a2b3c4a",  "start": "2023-05-01T15:00:00+00:00", "end" => "2023-05-01T16:00:00+00:00" }]',
```

Comment:
```suggestion
        example: [[ "period" => "/periods/1a2b3c4a",  "start" => "2023-05-01T15:00:00+00:00", "end" => "2023-05-01T16:00:00+00:00" ]],
```
Nice catch, but you are mixing `:` and `=>`, and we can write this as PHP objects to get a more accurate example value (array of objects rather than JSON string)

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/3176/commits/dfb0b16cef2304a2b95128b69eebea07f3699e82